### PR TITLE
Hydrogen is incompatible with Optifine/Optifabric; make this combo illegal

### PR DESCRIPTION
**Pull request by MeeniMc**
Make explicit the fact that Hydrogen and Optifine/Optifabric don't mix well. 

This is in response to issues #41, issue #43 where users got confused; and in acknowledgement that #20 is a 'wont fix'.

### CODE REVIEW

**Comment by arceuss** [comment]
inb4 people still make issues about both not working together

**Comment by K0-RR** [comment]
This won't help in all cases as some people don't understand what "found conflicting mods" mean.
https://github.com/Gegy/tic-tacs/issues/43

File: gradle.properties [hunk 1]
```diff
@@ -9,10 +9,10 @@ loader_version=0.11.6
 fabric_api_version=0.35.2+1.17
 
 # Mod Properties
-mod_version=0.3
+mod_version=0.3.1
```
Comment (jellysquid3) [inline gradle.properties]: Please don't bump versions in PRs.

**Review by jellysquid3 — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (MeeniMc) [reply gradle.properties]: Ok, I'll force push to remove this

**Review by 5HT2 — APPROVED** [review]

**Review by jellysquid3 — APPROVED** [review]

**Pull request merged**